WordPress/WordPress-Alpha.entitlements

```diff
@@ -11,7 +11,7 @@
 	</array>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>group.org.wordpress.alpha</string>
+		<string>group.com.jetpack.alpha</string>
```

Comment:
This is changing the `WordPress-Alpha` entitlement. Is that an unintentional change?

Reply:
That's odd – shouldn't have made it in. I deleted it, thanks!